test/parallel/test-vm-preserves-property.js

```diff
@@ -0,0 +1,23 @@
+var assert = require('assert');
```

Comment:
[usually](https://github.com/iojs/io.js/blob/v1.x/test/parallel/test-vm-basic.js#L1), tests have a require call to `../common`. could you add that here?